src/Command/Import_Command.php

```
@@ -0,0 +1,718 @@
+		// 'Importing: %s | Type: %s | Status: %s | URL: %s',
+		// $title,
+		// $post_type,
+		// $post_status,
+		// $content_scrapper->get_url()
+		// ) );
+		//
+		// ADD YOUR CUSTOM POST PROCESSING LOGIC HERE
+		// ====================================================================
+
+		// If creating a post.
+		$post = $inserter->create();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$post'. (Unused Code Rules)

(UnusedLocalVariable)